app/lib/pre_assembly/from_staging_location/file_set_builder.rb

```diff
@@ -35,14 +36,23 @@ def build
 
       attr_reader :processing_configuration, :objects, :style
 
+      # until the new OCR settings are available, we have to look in the processing configuration
+      def ocr_available
+        if Settings.ocr.enabled
+          @ocr_available
+        else
+          processing_configuration == :filename_with_ocr
+        end
+      end
+
```

Comment:
allows us to work with both processing configuration or radio button to determine if OCR is available for now...